apps/src/templates/CheckboxDropdown.jsx

```diff
@@ -15,25 +20,49 @@ const CheckboxDropdown = ({
       className="selectbox"
       data-toggle="dropdown"
     >
+      {checkedOptions.length > 0 && (
+        <FontAwesome id={'check-icon'} icon="check-circle" />
```

Comment:
Since this check icon indicates a meaning that doesn't have words associated with it, could you add a `title` prop explaining what this indicates? Maybe something like "Filter for {filter_key} applied" or something like that ... it would need to be translated.

Reply:
Good call! Will do, I'm thinking "{filter_label} filter applied" and using the label since the label will be translated (whereas the key won't be)

Reply:
+1 that makes sense! Sounds good 👍